xtask/codegen/src/generate_configuration.rs

```diff
@@ -141,7 +141,7 @@ pub(crate) fn generate_rules_configuration(mode: Mode) -> Result<()> {
         use biome_json_analyze::options::*;
         use biome_css_analyze::options::*;
         use biome_rowan::TextRange;
-        use indexmap::IndexSet;
+        use rustc_hash::FxHashSet;
```

Comment:
I noticed you're slowly removing `IndexSet` from the code base. Maybe we should do that in another PR; this feels more like a refactor, and unrelated to the PR. But it's fine for now.